site_root/ig.json

```diff
@@ -1,72 +1,88 @@
+    "defaults": {
+        "Any": {
+            "template-base": "source/templates/base.html",
+            "template-format": "source/templates/format.html"
+        },
+        "CodeSystem": {
+            "template-base": "source/templates/base.html"
+        },
+        "Composition": {
+            "template-base": "source/templates/ex.html"
+        },
+        "DocumentReference": {
+            "template-base": "source/templates/ex.html"
+        },
+        "FamilyHistory": {
+            "template-base": "source/templates/ex.html"
+        },
+        "Observation": {
+            "template-base": "source/templates/ex.html"
+        },
+        "OperationDefinition": {
+            "template-base": "source/templates/base.html"
+        },
+        "Patient": {
+            "template-base": "source/templates/ex.html"
+        },
+        "StructureDefinition": {
+            "template-base": "source/templates/sd.html",
+            "template-defns": "source/templates/sd-definitions.html"
+        },
+        "ValueSet": {
+            "template-base": "source/templates/base.html"
+        }
+    },
+    "fixed-business-version": "0.1.0",
+    "license": "CC0-1.0",
+    "npm-name": "kidsfirst.fhir",
     "pages": [
-      "framework",
-      "source/pages"
+        "pages"
     ],
-    "resources": [
-      "source/resources",
-      "source/examples"
-    ]
-  },
-  "pages": [
-    "pages"
-  ],
-  "sct-edition": "http://snomed.info/sct/900000000000207008",
-  "canonicalBase": "http://fhir.kids-first.io",
-  "defaults": {
-    "StructureDefinition": {
-      "template-base": "source/templates/sd.html",
-      "template-defns": "source/templates/sd-definitions.html"
-    },
-    "Patient": {
-      "template-base": "source/templates/ex.html"
-    },
-    "OperationDefinition": {
-      "template-base": "source/templates/base.html"
-    },
-    "CodeSystem": {
-      "template-base": "source/templates/base.html"
-    },
-    "Any": {
-      "template-format": "source/templates/format.html",
-      "template-base": "source/templates/base.html"
-    },
-    "ValueSet": {
-      "template-base": "source/templates/base.html"
-    },
-    "Observation": {
-      "template-base": "source/templates/ex.html"
-    },
-    "DocumentReference": {
-      "template-base": "source/templates/ex.html"
+    "paths": {
+        "output": "output",
+        "pages": [
+            "framework",
+            "source/pages"
+        ],
+        "qa": "generated_output/qa",
+        "resources": [
+            "source/resources",
+            "source/examples"
+        ],
+        "specification": "http://hl7.org/fhir/R4/",
+        "temp": "generated_output/temp",
+        "txCache": "generated_output/txCache"
     },
-    "FamilyHistory": {
-      "template-base": "source/templates/ex.html"
+    "resources": {
+        "Observation/phenotype-example": {
+            "base": "Observation-phenotype-example.html",
+            "source": "phenotype-example.json"
```

Comment:
This isn't really needed. The publisher will look for files by "id" if the "Type-id" fails to find the file. But, to remain consistent with the naming pattern, maybe this file should be named "Observation-phenotype-example.json", and this file name will be looked for first. Also, the "-example" suffix isn't really needed. Any file that starts with a Type name other than the conformance types is probably an example file, and will likely only be in the examples folder.